Documentation/ceph-prerequisites.md

```diff
@@ -57,6 +57,13 @@ runcmd:
 If you want to configure volumes with the Flex driver instead of CSI, the Rook agent requires setup as a Flex volume plugin to manage the storage attachments in your cluster.
 See the [Flex Volume Configuration](flexvolume.md) topic to configure your Kubernetes deployment to load the Rook volume plugin.
 
+## Troubleshooting Ceph-CSI
```

Comment:
Instead of this section, how about adding a link from ceph-common-issues.md? I don't think we need troubleshooting links on the prereqs page.